values-schema.yaml

```diff
@@ -172,8 +172,22 @@ definitions:
     pattern: '^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$'
     type: string
   env:
-    pattern: '^[a-zA-Z0-9_]*$'
-    type: string
+    items:
+      additionalProperties: false
+      properties:
+        name:
+          pattern: '^[a-zA-Z0-9_]*$'
+          type: string
+        value:
+          maxLength: 131072
+          type: string
+      required:
+        - name
+        - value
+      type: object
```

Comment:
please don't use verbose version and allow `key: value` format for easier input

Reply:
I just copied this block to a definition to make it more DRY, but I can definitely do it. So you want to include a refactor of this inside the scope?

Reply:
Again, please refactor this to use `key: value` format using `patternProperties`, as we allow all env vars (or anything actually) to be specified in a DRY fashion

Reply:
@svatwork has this been done?